### PR TITLE
Reformatted multiline doc strings to PEP 

### PR DESCRIPTION
**Pull request by mirob9363**
Addresses issue https://github.com/earthlab/earthpy/issues/207

### CODE REVIEW

File: earthpy/clip.py [hunk 1]
```diff
@@ -4,11 +4,10 @@
 
 
 def _clip_points(shp, clip_obj):
-    """ A function to clip point geometry using geopandas. Takes an
-    input point GeoDataFrame that will be clipped to the clip_obj
-    GeoDataFrame.
+    """Clip point geometry using geopandas.
```
Comment (lwasser) [inline earthpy/clip.py]: How about:
`Clip point geometry using a GeoDataFrame polygon extent.`

File: earthpy/clip.py [hunk 2]
```diff
@@ -4,11 +4,10 @@
...
 
-    Points that intersect with the geometry of clip_obj are extracted
+    Take an input point GeoDataFrame that will be clipped to the clip_obj
+    GeoDataFrame. Points that intersect with the geometry of clip_obj are extracted
     and returned.
```
Comment (lwasser) [inline earthpy/clip.py]: how about: 
`Clip an input point GeoDataFrame to the polygon extent of the clip_obj parameter. Points that intersect the clip_obj geometry are extracted with associated attributes and returned.`

File: earthpy/clip.py [hunk 3]
```diff
@@ -30,13 +29,11 @@ def _clip_points(shp, clip_obj):
 
 
 def _clip_line_poly(shp, clip_obj):
```
Comment (lwasser) [inline earthpy/clip.py]: I'd use the same language that you used above for points for the lines function!

File: earthpy/clip.py [hunk 4]
```diff
@@ -72,8 +69,7 @@ def _clip_line_poly(shp, clip_obj):
 
 
 def clip_shp(shp, clip_obj):
-    """A function to clip points, lines, polygon geometries based on an input
-    geometry.
+    """Clip points, lines, or polygon geometries.
```
Comment (lwasser) [inline earthpy/clip.py]: How about: 
`Clip points, lines, or polygon geometries to the clip_obj geometry.`

File: earthpy/clip.py [hunk 5]
```diff
@@ -30,13 +29,11 @@ def _clip_points(shp, clip_obj):
 
 
 def _clip_line_poly(shp, clip_obj):
-    """A function to clip line and polygon data using geopandas.
+    """Clip line and polygon data using geopandas.
```
Comment (lwasser) [inline earthpy/clip.py]: How about:
`Clip line and polygon geometries to the clip_obj GeoDataFrame geometry.`

File: earthpy/clip.py [hunk 6]
```diff
@@ -30,13 +29,11 @@ def _clip_points(shp, clip_obj):
...
```
Comment (lwasser) [inline earthpy/clip.py]: Actually i'm wondering if we need the longer docstring in this instance? the short one should be enough i think? @mbjoseph what do you think about these docstrings? keep is short and sweet? i'm not sure that people need the detail of us using spatial indices and such.

File: earthpy/mask.py [hunk 7]
```diff
@@ -87,8 +87,7 @@
 
 
 def _create_mask(mask_arr, vals):
```
Comment (lwasser) [inline earthpy/mask.py]: `Mask specific values in a 1-dimensional numpy array.`

File: earthpy/mask.py [hunk 8]
```diff
@@ -128,7 +127,7 @@ def _create_mask(mask_arr, vals):
 
 
 def _apply_mask(arr, input_mask):
-    """Applies a single dimension mask to the provided array.
+    """Apply a single dimension mask to the provided array.
```
Comment (lwasser) [inline earthpy/mask.py]: `Apply a mask to each band in the provided array.`

File: earthpy/mask.py [hunk 9]
```diff
@@ -161,10 +160,14 @@ def _apply_mask(arr, input_mask):
 
 
 def mask_pixels(arr, mask_arr, vals=None):
-    """Take an input array to be masked, single band mask layer such as a
-    pixel_qa layer for MODIS or Landsat, and apply a mask given a range of
-    values to mask. This function can also be passed a previously created
-    mask in place of a qa layer and a list of values.
+    """Apply a mask to an input array.
+
```
Comment (lwasser) [inline earthpy/mask.py]: How about:
`The mask_pixels function takes an input 1-dimensional masked array (mask_arr) and masks values in an n-dimensional input array. If mask_arr is provided in a boolean format, it is used as a mask. If mask_arr is provided as a non-boolean format and vals are provided, the function first masks values in the 1-dimensional mask_arr and then uses that mask_arr to mask the input arr. This function is useful when masking cloud and other unwanted pixels (e.g. pixel qa layers) found in landsat and other remote sensing data sets.   `

i dont love this text above btw but it's a start maybe?

File: earthpy/plot.py [hunk 10]
```diff
@@ -14,8 +14,9 @@
 
 
 def colorbar(mapobj, size="3%", pad=0.09):
-    """Adjusts the height of a colorbar to match the axis height. Note that
-    this function will not work properly using matplotlib v 3.0.0 in Jupyter
+    """Adjust colorbar height to match the axis height.
```
Comment (lwasser) [inline earthpy/plot.py]: How about
`Adjust colorbar height to match the matplotlib axis height.`

File: earthpy/plot.py [hunk 11]
```diff
@@ -14,8 +14,9 @@
...
+
```
Comment (lwasser) [inline earthpy/plot.py]: `NOTE: This function requires matplotlib v 3.0.1 or greater or  v 2.9 or lower to run properly. `

File: earthpy/plot.py [hunk 12]
```diff
@@ -74,9 +75,7 @@ def colorbar(mapobj, size="3%", pad=0.09):
 def plot_bands(
     arr, cmap="Greys_r", figsize=(12, 12), cols=3, title=None, extent=None
 ):
-    """Plot each layer in a raster stack read from rasterio in
-    (band, row , col) order as a numpy array. plot_bands will create an
-    individual plot for each band in a grid.
+    """Individually plot each band from rasterio order(band, row , col) stack.
```
Comment (lwasser) [inline earthpy/plot.py]: `Plot each band  in a numpy array in its own axis. Assumes band order (band, row, col)`

File: earthpy/plot.py [hunk 13]
```diff
@@ -175,7 +174,7 @@ def plot_bands(
 
 
 def _stretch_im(arr, str_clip):
-    """Stretch an image array in numpy format using a specified clip value
+    """Stretch an image array using a specified clip value.
```
Comment (lwasser) [inline earthpy/plot.py]: `Stretch an image in numpy ndarray format using a specified clip value.`

File: earthpy/plot.py [hunk 14]
```diff
@@ -386,9 +385,10 @@ def hist(
 
 def make_col_list(unique_vals, nclasses=None, cmap=None):
     """
-    Take a defined matplotlib colormap, and create a list of colors based on
-    a set of values. This is useful when you need to plot a series of
-    classified numpy arrays that are missing some of the sequential classes.
+    Take a matplotlib colormap and create a list of colors.
```
Comment (lwasser) [inline earthpy/plot.py]: How about: 
`Convert a matplotlib named colormap into a discrete list of n-colors in RGB format.`

I think you can remove the rest of the text then !

File: earthpy/spatial.py [hunk 15]
```diff
@@ -57,9 +58,7 @@ def extent_to_json(ext_obj):
 
 
 def normalized_diff(b1, b2):
```
Comment (lwasser) [inline earthpy/spatial.py]: How about
`Take two n-dimensional numpy arrays and calculate the normalized difference.`

`Math will be calculated (b1-b2) / (b1 + b2). `

File: earthpy/spatial.py [hunk 16]
```diff
@@ -117,9 +116,7 @@ def normalized_diff(b1, b2):
 
 # TODO: include a no data value here if provided
 def stack(band_paths, out_path=""):
-
-    """Take a list of raster paths and turn it into an output raster stack
-    numpy array.
+    """Convert a list of raster paths into a raster stack numpy array.
```
Comment (lwasser) [inline earthpy/spatial.py]: `Convert a list of raster paths into a stacked numpy ndarray.`

**Comment by mirob9363** [comment]
@lwasser I have updated the doc strings based off your suggestions. Let me know what you think!

File: earthpy/plot.py [hunk 17]
```diff
@@ -444,8 +448,25 @@ def draw_legend(im_ax, bbox=(1.05, 1), titles=None, cmap=None, classes=None):
...
+    >>> im_arr_bin = np.digitize(im_arr, bins)
+    >>> cat_names = ["Enhanced Regrowth",
+                  "Unburned",
+                  "Low Severity",
+                  "Moderate Severity",
+                  "High Severity"]
+    >>> f, ax = plt.subplots()
+    >>> im = ax.imshow(im_arr_bin, cmap="gnuplot")
+    >>> im_ax = ax.imshow(im_arr_bin)
+    >>>> leg_neg = ep.draw_legend(im_ax = im_ax, titles = cat_names)
```
Comment (lwasser) [inline earthpy/plot.py]: @mirob9363 it looks like this line has one extra `>`

**Comment by lwasser** [comment]
@mirob9363 notice that this PR is failing continuous integration tests. I found one issue - you have an extra `>>>` please be sure that the build is passing travis before requesting another review! you can build the docs locally to see what the issues are as well!!

**Comment by lwasser** [comment]
fixed all of the issues! merging! thank you @mirob9363 !!

**Pull request merged**